application/src/main/java/org/opentripplanner/apis/transmodel/TransmodelGraphQLSchema.java

```diff
@@ -1287,7 +1289,7 @@ private GraphQLSchema create() {
             GraphQLArgument
               .newArgument()
               .name("activeDates")
-              .description("Set of ids of active dates to fetch serviceJourneys for.")
+              .description("Set of operating-days to fetch service-journeys for.")
```

Comment:
This was @t2gran's suggestion. I guess operatingDays or operating days would be alternatives?
<!-- Reviewable comment -OBUcjL_216H_3EOPF-c:b-8qm78c -->
<!-- Sent from Reviewable.io -->

Reply:
We can talk about it on the developer meeting. I think in regular English it would be written as "operating days" without a hyphen. And if we want to reference a specific concept OTP maybe we should use the same spelling as in the code `OperatingDay` or whatever we are using.

I don't have a strong preference other than to keep it consistent within the codebase.

Reply:
At the meeting we said that we should prefer markdown emphasis in this case since it's markdown.

Reply:
We discussed this in the developer meeting today, this is the conclusion:

When referencing established OTP terminology we should:
 - Use _emphizise_ "\_dated service journey\_" in markdown documentation, in API and configuration documentation.
 - Use hyphen "dated-service-journey" in plain text and JavaDoc. In JavaDoc we also use {@link DatedServiceJourney} the first time we mention a class. Note that sometimes we want to talk about the concept(use hyphens) and sometime reference a class(use @link).